flutter/lib/src/widgets_binding_observer.dart

```diff
@@ -20,11 +24,31 @@ class SentryWidgetsBindingObserver with WidgetsBindingObserver {
     Hub? hub,
     required SentryFlutterOptions options,
   })  : _hub = hub ?? HubAdapter(),
-        _options = options;
+        _options = options,
+        _screenSizeStreamController = StreamController(sync: true) {
+    if (_options.enableWindowMetricBreadcrumbs) {
+      _screenSizeStreamController.stream
+          .map(
+            (window) => {
+              'new_pixel_ratio': window?.devicePixelRatio,
+              'new_height': window?.physicalSize.height,
+              'new_width': window?.physicalSize.width,
+            },
+          )
+          .distinct(mapEquals)
+          .skip(1) // Skip initial event added below in constructor
```

Comment:
It's needed to make it emit events only when they actually changed. I mean, if at the beginning screen size was X, and then user opened a keyboard, screen size is still X, but if distinct stream is empty, it will emit `Screen size changed` event, though it actually didn't, because it had nothing to compare with. So i've added initial configuration, prevented it from emitting due to `.skip(1)`, and it works as expected: new events will come only when screen size or pixel ratio actually changes  It's mainly needed to match to the current test suite

It also fits to the actual test suite